.ci/snap_publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+SNAPCRAFT_BUILD_ENVIRONMENT_CPU=$(nproc)
+export SNAPCRAFT_BUILD_ENVIRONMENT_CPU
+SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=16G
+export SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY
+
+# Prevent running if we've already published this revision
+apt update
+apt install git -y
```

Comment:
From the manual,

> All features of apt(8) are available in dedicated APT tools like apt-get(8) and apt-
       cache(8) as well.  apt(8) just changes the default value of some options (see apt.conf(5)
       and specifically the Binary scope). So you should prefer using these commands (potentially
       with some additional options enabled) in your scripts as they keep backward compatibility
       as much as possible.

Reply:
I wouldn't expect usage / configuration of these commands to change in a way that would break things though. Just good to consider.

Reply:
I don't expect it to change either, but if I'm doing something more complicated I'll make sure to use the proper commands.